src/parsers/transactions.ts

```diff
@@ -290,13 +284,54 @@ const overrideTradeRefund = (txn: ZerionTransaction): ZerionTransaction => {
   };
 };
 
-const parseTransaction = (
+const parseTransactionWithEmptyChanges = async (
+  txn: ZerionTransaction,
+  nativeCurrency: string
+) => {
+  const methodName = await getTransactionMethodName(txn);
+  const updatedAsset = {
+    address: ETH_ADDRESS,
+    decimals: 18,
+    name: 'ethereum',
+    symbol: 'ETH',
+  };
+  const priceUnit = 0;
+  const valueUnit = 0;
+  const nativeDisplay = convertRawAmountToNativeDisplay(
+    0,
+    18,
+    priceUnit,
+    nativeCurrency
+  );
+
+  return [
+    {
+      address: ETH_ADDRESS,
+      balance: convertRawAmountToBalance(valueUnit, updatedAsset),
+      description: methodName || 'Contract interaction',
```

Comment:
Gonna confirm with Christian what happens in the case that methodName does not exist - seems weird to have "Contract interaction" as both the title and the description